src/chain/chain.rs

```diff
@@ -432,7 +444,7 @@ impl Chain {
         self.state.our_prefix()
     }
 
-    /// Returns whether our section is in the process of changing (splitting or merging).
+    /// Returns whether our section is in the process of changing (splitting, merging or pruning).
     pub fn prefix_change(&self) -> PrefixChange {
```

Comment:
Does this method name now needs updating to reflect the new responsibility?

Reply:
This follows the enum name. unless we want change them all ?

Reply:
Yeah that's a good question. I think some of these names needs updating at least. But maybe we can do that once more pieces are in place?